tests/text.test.js

```diff
@@ -28,4 +34,27 @@ describe('Text', () => {
     expect(text.layoutEngine.layout.mock.calls).toHaveLength(1);
     expect(text.layoutEngine.layout.mock.calls[0][0].string).toBe('');
   });
+
+  test('Should render the same rect x value for clones', async () => {
```

Comment:
I dont love how this test is constructed. Let me know if you have a better means to test this!

Reply:
I think is fine! Maybe there's a simpler way, but this is not bad either. Thank you very much!